bash/start_hdp.sh

```diff
@@ -1942,8 +1942,12 @@ function f_dockerfile() {
     f_ssh_setup
 
     local _pkey="`sed ':a;N;$!ba;s/\n/\\\\\\\n/g' $HOME/.ssh/id_rsa`"
+    local _pubkey="`sed ':a;N;$!ba;s/\n/\\\\\\\n/g' $HOME/.ssh/authorized_keys`"
```

Comment:
i didn't test but I think this doesn't guarantee to return one or correct pubkey 
maybe simply copying authorized_keys to all containers would be easier?